src/auto/accel_group.rs

```diff
@@ -49,11 +53,11 @@ impl Default for AccelGroup {
 pub trait AccelGroupExt {
     //fn activate<P: IsA<glib::Object>>(&self, accel_quark: /*Ignored*/glib::Quark, acceleratable: &P, accel_key: u32, accel_mods: gdk::ModifierType) -> bool;
 
-    //fn connect(&self, accel_key: u32, accel_mods: gdk::ModifierType, accel_flags: AccelFlags, closure: /*Ignored*/&glib::Closure);
+    fn connect(&self, accel_key: u32, accel_mods: gdk::ModifierType, accel_flags: AccelFlags, closure: &glib::Closure);
```

Comment:
Same here

Reply:
And basically everywhere where a `GClosure` appears but the API actually expects a specific signature